tests/test_auxtel_latiss_take_sequence.py

```diff
@@ -0,0 +1,176 @@
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import contextlib
+import types
+import unittest
+import logging
+
+import pytest
+from lsst.ts import salobj
+from lsst.ts.standardscripts.auxtel import LatissTakeSequence
+from lsst.ts.standardscripts import BaseScriptTestCase, get_scripts_dir
+
+
+logger = logging.getLogger(__name__)
+logger.propagate = True
+
+
+class TestLatissTakeSequence(BaseScriptTestCase, unittest.IsolatedAsyncioTestCase):
+    def setUp(self) -> None:
+        self.ataos_hexapod_correction_status = True
+        self.ataos_m1_correction_status = True
+        self.ataos_atspectrograph_correction_status = True
+        return super().setUp()
+
+    async def basic_make_script(self, index):
+        self.script = LatissTakeSequence(index=index, add_remotes=False)
+
+        self.end_image_tasks = []
+
+        # things to track
+        self.nimages = 0
+        self.date = None  # Used to fake dataId output from takeImages
+        self.seq_num_start = None  # Used to fake proper dataId from takeImages
+
+        logger.debug("Finished initializing from basic_make_script")
+
+        return [
+            self.script,
+        ]
+
+    async def get_evt_correctionEnabled(self, timeout):
+        return types.SimpleNamespace(
+            hexapod=self.ataos_hexapod_correction_status,
+            m1=self.ataos_m1_correction_status,
+            atspectrograph=self.ataos_atspectrograph_correction_status,
+        )
+
+    @contextlib.asynccontextmanager
+    async def setup_mocks(self):
+        self.script.latiss.take_object = unittest.mock.AsyncMock()
+        self.script.atcs.check_tracking = unittest.mock.AsyncMock()
+        self.script.atcs.stop_tracking = unittest.mock.AsyncMock()
+        self.script.atcs.disable_ataos_corrections = unittest.mock.AsyncMock()
+
+        self.script.atcs.rem = types.SimpleNamespace(ataos=unittest.mock.AsyncMock())
+        self.script.atcs.rem.ataos.configure_mock(
+            **{
+                "evt_correctionEnabled.aget.side_effect": self.get_evt_correctionEnabled,
+            }
+        )
+        yield
+
+    async def test_executable(self):
+        scripts_dir = get_scripts_dir()
+        script_path = scripts_dir / "auxtel" / "latiss_take_sequence.py"
+        logger.debug(f"Checking for script in {script_path}")
+        await self.check_executable(script_path)
+
+    async def test_configure(self):
+        async with self.make_script(), self.setup_mocks():
```

Comment:
Can you separate the different tests into separate test methods with names that explain what you are testing?

Basically, put every `async with self.make_script(), self.setup_mocks():`  in a different test function.